config/model-unet.toml

```diff
@@ -29,3 +29,6 @@
 
   # Weight decay l2 penalty for the optimizer
   decay      = 0.0001
+
+  # Loss function name (e.g 'CrossEntropy', 'mIoU', 'Lovasz' or 'Focal')
+  loss = 'CrossEntropy'
```

Comment:
Should we support weighted losses? E.g. a * cross entropy + b * dice? I think that was the initial plan in https://github.com/mapbox/robosat/issues/3. Do you think it makes sense to let user specify that?

I would love if we could have sensible default settings users without domain knowledge in losses should tweak. I'm not sure cross entropy is a good default choice now that we have lovasz for example. Did you run experiments and do you have numbers we can compare?

Also the focal loss should probably not be used until we implement retina net for object detection, and even then we need to benchmark it first.

Reply:
Related to #3

After several attempts (on mIoU)  i finally find that a max(loss1 , loss2) could be an easy way to reach good results with fewer hyperparameters to adjust.
Dice based loss imply to find a good enough tractable approximation, and did'nt take time to look on this at this point.

You're right on the easy to play configuration, and
i just switch from CrossEntropy to Lovasz default loss.

And if we talk about numbers Lovasz and mIoU are close.
They both better handle shape segmentation on near features and reduce probs noise. 
We could even imagine removing one step in the process as probs and masks output are really similars.

X Entropy: mIoU: 0.835, building IoU: 0.735
mIoU:        mIoU: 0.848, building IoU: 0.752 
Lovasz:     mIoU: 0.853, building IoU: 0.759

Reply:
![107930](https://user-images.githubusercontent.com/1718264/45884132-53f28b00-bdb3-11e8-8f44-cc77457a95f9.png)

One compare probs predict sample: ( image / label / x_entropy / mIoU / Lovasz )

Reply:
Fascinating! Amazing work! :tada: